at this point this is my theme with cool as heck blood moon wallpaper
probably gotta swap colors a bit, but palette is ok for most of the stuff
thanks, terminal.sexy
---
 config.def.h | 44 ++++++++++++++++++++++----------------------
 1 file changed, 22 insertions(+), 22 deletions(-)

diff --git a/config.def.h b/config.def.h
index 0e01717..fb504d4 100644
--- a/config.def.h
+++ b/config.def.h
@@ -85,30 +85,30 @@ unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#000000"
+	"#670604"
+	"#941404"
+	"#c03e04"
+	"#af2404"
+	"#d85605"
+	"#e44c04"
+	"#dddddd"
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#222222"
+	"#370404"
+	"#470404"
+	"#d04504"
+	"#bf2f04"
+	"#fa8b05"
+	"#f9a151"
+	"#ffffff"
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#000000", /* background */
+	"#ffffff", /* foreground */
 };
 
 
@@ -116,10 +116,10 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+static unsigned int defaultcs = 257;
+static unsigned int defaultrcs = 256;
 
 /*
  * Default shape of cursor
-- 
2.22.0

